### PR TITLE
Implements exome-seq workflow

### PR DESCRIPTION
**Pull request by dleehr**
The workflow is still in development (see https://trello.com/c/AhdNNh1o/80-bespin-exome-seq-workflow), but this PR will be referenced by issues so we know that they're addressed

### CODE REVIEW

**Comment by dleehr** [comment]
Fixes #6

**Comment by dleehr** [comment]
Fixed #7

**Comment by dleehr** [comment]
Fixes #13 and #12

**Comment by dleehr** [comment]
Fixes #8, #9 and #14

File: workflows/exomeseq.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Exome-seq workflow
+
+Based on https://software.broadinstitute.org/gatk/best-practices/bp_3step.php?case=GermShortWGS
+
+Data Sets:
+
+- Reference genome in uncompressed .fa format (GATK does not accept the compressed version) For human genome, we use broad b37 bundle. This is idnexed with bwa, and different steps require different secondaryFiles for its index.
```
Comment (johnbradley) [inline workflows/exomeseq.md]: idnexed -> indexed

File: workflows/exomeseq.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Notes:
+
+- Broad recommends raw illumina basecalls, converted to unmapped SAM. This has more metadata, but is larger and won't typically be available from sequencing facilities like FASTQ. May be possible in more integrated facilities or in the future, but it not available now.
```
Comment (johnbradley) [inline workflows/exomeseq.md]: `but it not available now.` -> `but it is not available now.`

**Comment by dleehr** [comment]
Thanks!

**Pull request merged**